xaod_cpp_transformer/requirements.txt

```diff
@@ -10,7 +10,9 @@ pyarrow==0.16.0
 pytest==4.6.11
 pytest-mock==2.0.0
 requests==2.24.0
-servicex-transformer==1.0.3
-uproot==3.11.7
+# servicex-transformer==1.0.3
```

Comment:
Any reason this can't just be 1.0.4? Why are we depending on an RC of this library?